dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/master/runner/WorkflowExecuteThread.java

```diff
@@ -480,28 +481,45 @@ private boolean processComplementData() throws Exception {
                     processInstance.getScheduleTime(),
                     complementListDate.toString());
             scheduleDate = complementListDate.get(index + 1);
-            //the next process complement
-            processInstance.setId(0);
+
+        }
+
+        this.taskInstanceHashMap.clear();
+
+        //the next process complement
+        int create = this.createComplementDataCommand(scheduleDate);
+        if (create > 0) {
+            logger.info("create complement data command successfully.");
```

Comment:
It's better to add some identify flag in log, such as id or code.

Reply:
I'll add it.